types/container/host_config.go

```diff
@@ -216,6 +216,7 @@ type HostConfig struct {
 	GroupAdd        []string           // List of additional groups that the container process will run as
 	IpcMode         IpcMode            // IPC namespace to use for the container
 	Links           []string           // List of links (in the name:alias form)
+	Aliases         []string           // List of aliases
```

Comment:
Actually. I reviewed what has been done in https://github.com/docker/docker/pull/19001 and I think that make sense.
If we go with the `NetworkingConfig` approach, then we can remove this field from HostConfig.
Since the Aliases is carried in `EndpointSettings` already.